components/frontend/src/header_footer/buttons/DownloadAsPDFButton.test.jsx

```diff
@@ -56,7 +56,7 @@ test("DownloadAsPDFButton indicates loading on click", async () => {
     expectButtonIsLoading()
     expect(get_report_pdf.get_report_pdf).toHaveBeenCalledWith(
         "report_uuid",
-        "?report_url=http%3A%2F%2Flocalhost%3A3000%2F",
+        "?language=en-US&report_url=http%3A%2F%2Flocalhost%3A3000%2F%26language%3Den-US",
```

Comment:
Should we also autotest `nl-NL` somewhere?

Reply:
Agreed. I'll see if I can mock `navigator.language`.